providers/sru/units/sru-ubuntucore.pxu

```diff
@@ -39,7 +39,7 @@ include:
     ethernet/detect
     ethernet/info_automated
     networking/http
-    ethernet/ping_.*
+    ethernet/ping-with-any-cable-interface
```

Comment:
Ok I can see that this may be very controversial. Are we sure we don't want to test all interfaces? This reduces the coverage of the test suite quite a bit

Reply:
ok, won't we ever have the need of testing all (/some) interfaces? I'm a bit worried we may have a a regression on some interfaces but not others. Is this something that could/may happen? It is a bit strange that we don't exclude some interface from the testing instead of just running only the test on one

Reply:
Until we set up devices in the lab in a way that every port is connected, this is the only way I'm affraid. (after reprovisioning the interface names may change, Pierre reported that).